GitExtensions.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.27130.2026
 MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
💡 You can set the minimum version here to 15.5 and it will prevent the solution from opening in a version of VS lower than the first one that supports C# 7.2. The improved error message experience can be seen in https://github.com/Microsoft/perfview/pull/319#issuecomment-318511183.

Reply:
Do you know if the revision/build values are important? Can we just use `15.5` or `15.5.5`?

Reply:
It should work fine to use `15.5` 👍

Reply:
`MinimumVisualStudioVersion = 15.5` gives me an error, as does 15.5.5.

![image](https://user-images.githubusercontent.com/350947/36548785-1618ea38-17e9-11e8-8451-4735f31f0aa6.png)

I have 15.5.5 installed. Not sure how to apply your suggestion here.

Reply:
Oh, duh, one second...

Reply:
15.5 is internally recognized as **15.0.27130.0**, so try that 😁